src/PollingBlockTracker.test.ts

```diff
@@ -392,60 +393,20 @@ describe('PollingBlockTracker', () => {
             },
           },
           async ({ blockTracker }) => {
-            const promiseForCaughtError = new Promise<any>((resolve) => {
-              blockTracker[methodToAddListener]('error', resolve);
-            });
+            const errorListener = jest.fn();
+            blockTracker[methodToAddListener]('error', errorListener);
 
             const promiseForLatestBlock = blockTracker.getLatestBlock();
 
-            const caughtError = await promiseForCaughtError;
-            expect(caughtError.message).toMatch(
-              /^PollingBlockTracker - encountered an error while attempting to update latest block:\nError: boom\n/u,
-            );
-            const latestBlock = await promiseForLatestBlock;
-            expect(latestBlock).toBe('0x0');
-          },
-        );
-      });
-
-      it(`should emit the "error" event (added via \`${methodToAddListener}\`) and should not throw if, while making the request for the latest block number, the provider throws a string`, async () => {
```

Comment:
Previously we had tests which verified that the error message produced when the thrown error was a string was different from when the thrown error was an Error. Because we've simplified how error messages are stringified, we don't need to do this anymore.